apps/calendar/js/views/month_day.js

```diff
@@ -76,11 +76,11 @@ MonthDay.prototype = {
     var holder = this.element.querySelector('.busy-indicator');
 
     if (count > 0) {
-      holder.setAttribute('aria-label', navigator.mozL10n.get('busy', {
-        n: count
-      }));
+      holder.setAttribute('data-l10n-id', 'busy');
+      holder.setAttribute('data-l10n-args', JSON.stringify({n: count}));
     } else {
-      holder.removeAttribute('aria-label');
```

Comment:
Actually lets keep ```holder.removeAttribute('aria-label');``` I believe the label is not cleared when you delete data-l10n attributes.